CONTRIBUTING.md

```diff
@@ -114,7 +114,7 @@ Feel free to add new benchmark tests if the current benchmark coverage is insuff
 
 When building new APIs, think about what it would be like to use them. Would this API cause confusing and hard to pin error mesages? Would this API integrate well with other APIs? Is it intuitive to use this API?
 
-There are many resources which provide good guidance on how to write APIs, a few of which are given below. These are only _guidelines_ and while they are useful and should be followed where possible, in some cases it may not be possible to do so.
+Explore useful guidance on how to write APIs below. These are only _guidelines_ and while they are helpful and should be followed where possible, in some cases, it may not be possible to do so.
```

Comment:
This doesn't read very well in my opinion. Can you think of a better way to phrase this?

Reply:
Let me think about it.

Reply:
@siku2 I changed it to:

Below, you can find some useful guidance and best practices on how to write APIs. These are only _guidelines_ and while they are helpful and should be followed where possible, in some cases, it may not be possible to do so.

Let me know your thoughts.